### PR TITLE
Update OrderedDataStore.yaml

### PR DESCRIPTION
**Pull request by cdyring**
Using SetAsync() with three parameters returns: DataStoreService: ParameterNotAllowed: Additional parameter UserIds not allowed. API: SetAsync, Data Store: XXXXXXX  -  Studio

I assume this also applies for increment.

## Changes

Added a short statement that supplying the UserIds parameter is not allowed when using SetAsync and IncrementAsync for ordered data stores. 

N/A

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/reference/engine/classes/OrderedDataStore.yaml [hunk 1]
```diff
@@ -16,6 +16,9 @@ description: |
   `Class.OrderedDataStore`. If you need versioning and metadata support, use a
   `Class.DataStore`.
 
+  Ordered data stores do not support the optional UserIDs parameter for
```
Comment (aetter) [inline content/en-us/reference/engine/classes/OrderedDataStore.yaml:19]:
```suggestion
  Ordered data stores do not support the optional `userIds` parameter for 
```

**Comment by aetter** [comment]
Hey @cdyring, this looks great. If you check the boxes, I'll get it merged ASAP. 
Screenshot 2024-05-16 at 4 47 03 PM

**Comment by cdyring** [comment]
Added the checkmarks

**Review by aetter — APPROVED** [review]

**Comment by aetter** [comment]
Thanks!

**Pull request merged**